src/disk.py

```diff
@@ -22,6 +23,7 @@
 STATE_FILENAME = "state.json"
 
 
+@tracer.wrap()
```

Comment:
word of caution--this function will be run a *lot,* like several times a minute. besides potential slowdowns, this might introduce a lot of noise to the tracing dashboard.

Reply:
Good point. Happy to remove it especially from functions that you're not worried about being slow

Reply:
yup, i'd delete. fwiw this will also be the case for something like `add_gtfs_headways` but there's enough complexity in that function that it could be still be worth tracing.